api/IapiFunctions.ts

```diff
@@ -0,0 +1,13 @@
+export interface IAccountData {
```

Comment:
Curious as to why this file should live here and not in the root directory in an interface folder?

i.e.: `src -> interfaces -> IapiFunctions.ts`

Reply:
This interface directly comes from the API. We should group common types together from where they are coming from. In this case, this is all API data.